Mixin not found crash

I launched the game with just this installed (Version 2.0.1 from CF), and a crash occurs due to a mixin not being found. It looks like the `fabric-biome-api.mixins.json` contains mixins which aren't used any more, which causes it to crash.

This was the error from the logs:
```
Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: The specified mixin 'net.fabricmc.fabric.mixin.biome.modification.BiomeAccessor' was not found
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinInfo.<init>(MixinInfo.java:864)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinConfig.prepareMixins(MixinConfig.java:850)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinConfig.prepare(MixinConfig.java:775)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:539)
	... 39 more
Caused by: java.lang.ClassNotFoundException: The specified mixin 'net.fabricmc.fabric.mixin.biome.modification.BiomeAccessor' was not found
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinInfo.loadMixinClass(MixinInfo.java:1313)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinInfo.<init>(MixinInfo.java:857)
	... 42 more
```

--- a/src/main/resources/fabric-biome-api.mixins.json
+++ b/src/main/resources/fabric-biome-api.mixins.json
@@ -1,22 +1,14 @@
 {
   "required": true,
   "package": "net.fabricmc.fabric.mixin.biome",
-  "compatibilityLevel": "JAVA_8",
+  "compatibilityLevel": "JAVA_16",
   "mixins": [
-    "modification.BiomeAccessor",
-    "modification.BiomeEffectsAccessor",
-    "modification.BiomeWeatherAccessor",
-    "modification.BiomeWeatherAccessor",
     "modification.DynamicRegistryManagerImplMixin",
-    "modification.GenerationSettingsAccessor",
-    "modification.RegistryOpsMixin",
-    "modification.SpawnDensityAccessor",
-    "modification.SpawnSettingsAccessor",
-    "MixinTheEndBiomeSource",
-    "MultiNoiseBiomeSourceAccessor"
+    "modification.MinecraftServerMixin",
+    "MixinMultiNoiseBiomeSource",
+    "MixinTheEndBiomeSource"
   ],
   "injectors": {
     "defaultRequire": 1
-  }, 
-  "minVersion": "0.8"
+  }
 }

